.travis.yml

```diff
@@ -24,11 +24,35 @@ jobs:
       script: npm run test && npm run e2e-test
       node_js: "8"
     - stage:
```

Comment:
Can this stage be named?

Reply:
When `stage` is empty, it means that it's in the same stage as the previously named `stage` and gets run concurrently with it.

Reply:
Oh yeah, I knew that.